.env

```diff
@@ -1,2 +1,3 @@
 VUE_APP_HUB_URL=https://testnet.snapshot.page
 VUE_APP_IPFS_GATEWAY=gateway.ipfs.io
+VUE_APP_IPFS_NODE=ipfs.io
```

Comment:
You don't need this, there is already `VUE_APP_IPFS_GATEWAY` that does the same, i just noticed that it's not used in the mixin, could you change the config var there?